build/pr-pipeline.yml

```diff
@@ -371,37 +276,18 @@ stages:
       keyVaultName: $(DeploymentEnvironmentNameR5)
       appServiceName: $(DeploymentEnvironmentNameR5)
 
-- stage: cleanupAks
-  displayName: 'Cleanup AKS Deployments'
-  dependsOn:
-  - testAksR4Cosmos
-  - testAksR4Sql
-  - testAksR4SqlContainer
-  jobs:
-  - template: ./jobs/cleanup-aks.yml
-    parameters:
-      subscription: $(ConnectedServiceName)
-      clusterName: $(clusterName)
-      clusterResourceGroup: $(clusterResourceGroup)
-
 - stage: cleanup
   displayName: 'Cleanup Azure Environment'
   dependsOn:
   - testStu3
   - testR4
   - testR5
-  - testAksR4Cosmos
-  - testAksR4Sql
-  - testAksR4SqlContainer
   condition: |
```

Comment:
This shouldn't be needed anymore. The default for dependsOn is that they all succeeded, this section was needed because the aks tests were being skipped.